AdminWebsite/AdminWebsite/ClientApp/src/app/shared/page-url.constants.ts

```diff
@@ -16,6 +16,5 @@ export let PageUrls = {
     DeleteParticipant: '/delete-participant',
     EditParticipantSearch: '/edit-participant-search',
     EditParticipant: '/edit-participant',
-    Login: '/login',
-    JoinLink: '/quickjoin'
+    Login: '/login'
```

Comment:
just double checking this wasnt removed by mistake? Dont we need the "quickjoin" link for the new endpoint

Reply:
they needed originally for the scan readers but BAs have confirmed it is not need at all as we copy a link to the clipboard directly.
This is the reason for this last change